src/qt/utilitydialog.cpp

```diff
@@ -33,29 +33,46 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
     ui(new Ui::HelpMessageDialog)
 {
     ui->setupUi(this);
-
+    this->resize(500,0);//force view height to stretch
+    ui->aboutMessage->resize(0,0);//force label to stretch
     QString version = QString{PACKAGE_NAME} + " " + tr("version") + " " + QString::fromStdString(FormatFullVersion());
 
-    if (about)
-    {
-        setWindowTitle(tr("About %1").arg(PACKAGE_NAME));
-
         std::string licenseInfo = LicenseInfo();
-        /// HTML-format the license message from the core
+        std::string copyrightInfo = CopyrightInfo();
+        // HTML-format aboutMessage elements
+        QString versionInfoHTML = QString::fromStdString(FormatVersion());
+        QString copyrightInfoHTML = QString::fromStdString(CopyrightInfo());
+        QString aboutMessageInfoHTML = QString::fromStdString(AboutMessageInfo());
         QString licenseInfoHTML = QString::fromStdString(LicenseInfo());
         // Make URLs clickable
         QRegExp uri("<(.*)>", Qt::CaseSensitive, QRegExp::RegExp2);
         uri.setMinimal(true); // use non-greedy matching
+        versionInfoHTML.replace(uri, "<a href=\"\\1\">\\1</a>");
+        //copyrightInfoHTML.replace(uri, "<a href=\"\\1\">\\1</a>");
+        aboutMessageInfoHTML.replace(uri, "<a href=\"\\1\">\\1</a>");
         licenseInfoHTML.replace(uri, "<a href=\"\\1\">\\1</a>");
         // Replace newlines with HTML breaks
+        copyrightInfoHTML.replace("\n", "<br>");
+        aboutMessageInfoHTML.replace("\n", "<br>");
         licenseInfoHTML.replace("\n", "<br>");
 
+        ui->copyrightInfo->setTextFormat(Qt::RichText);
         ui->aboutMessage->setTextFormat(Qt::RichText);
-        ui->scrollArea->setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
-        text = version + "\n" + QString::fromStdString(FormatParagraph(licenseInfo));
-        ui->aboutMessage->setText(version + "<br><br>" + licenseInfoHTML);
+        ui->licenseInfo->setTextFormat(Qt::RichText);
+        ui->versionInfo->setTextFormat(Qt::RichText);
         ui->aboutMessage->setWordWrap(true);
+        ui->scrollArea->setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
+        ui->copyrightInfo->setText(copyrightInfoHTML); //about bottom left corner
+        ui->aboutMessage->setText(aboutMessageInfoHTML); //aboutMessage bodyy
+        ui->licenseInfo->setText(licenseInfoHTML);
+        ui->versionInfo->setText(versionInfoHTML);
+
+
+    if (about)
+    {
+        setWindowTitle(tr("About %1").arg(PACKAGE_NAME));
```

Comment:
Over riding PACKAGE_NAME is outside the scope of this PR.

This PR is implementing changes based on broader community discussions and sentiment.

REF: https://github.com/bitcoin-core/gui/issues/89#issuecomment-705384143

REF: https://github.com/bitcoin-core/gui/pull/140#issuecomment-737773556